Makefile

```diff
@@ -1,8 +1,13 @@
+XCODE_MAJOR=`xcodebuild -version | head -n 1 | cut -f 2 -d' ' | cut -d. -f1`
+XCODE_MINOR=`xcodebuild -version | head -n 1 | cut -f 2 -d' ' | cut -d. -f2`
+XCODE_MICRO=`xcodebuild -version | head -n 1 | cut -f 2 -d' ' | cut -d. -f3`
+XCODE_VERSION=$(shell printf '%d' $(XCODE_MAJOR))$(shell printf '%02d' $(XCODE_MINOR))$(shell printf '%02d' $(XCODE_MICRO))
```

Comment:
Would it make sense to `export` the `XCODE_VERSION` definition? Then it could be reused inside Python using `os.environ['XCODE_VERSION']` without having to parse Xcode version again in Python. That would also make it impossible for the shell and Python `XCODE_VERSION` to have different values.